src/Microsoft.Health.Fhir.R4.Api/Features/Operations/OperationsCapabilityProvider.cs

```diff
@@ -0,0 +1,74 @@
+// -------------------------------------------------------------------------------------------------
```

Comment:
I don't think this file was part of the GraphQL work.

Reply:
**This file is not part of the GraphQL work**, while doing a merge with the main branch, github took the new files from others as my changes.